packages/muelu/adapters/xpetra/MueLu_RefMaxwell_def.hpp

```diff
@@ -1786,8 +1786,8 @@ namespace MueLu {
 
         NullspaceH_ = MultiVectorFactory::Build(P11_->getDomainMap(), dim);
 
-        auto localNullspace_nodal = Nullspace_nodal->getDeviceLocalView();
-        auto localNullspaceH = NullspaceH_->getDeviceLocalView();
+        auto localNullspace_nodal = Nullspace_nodal->getDeviceLocalView(Xpetra::Access::ReadOnly);
+        auto localNullspaceH = NullspaceH_->getDeviceLocalView(Xpetra::Access::ReadWrite);
```

Comment:
Write-only?

Reply:
I was not sure if all values are written or if some are just assumed to remain 0, can you confirm that 0s are written?

Reply:
Everything should be initialized to 0 when the MV is created here: https://github.com/trilinos/Trilinos/blob/d8c96d1ea19b5e60c1c6bc1b12c311f8b29b0ca0/packages/muelu/adapters/xpetra/MueLu_RefMaxwell_def.hpp#L1787
Entries that aren't modified are supposed to stay zero.